server/src/main/java/com/objectcomputing/checkins/services/checkinnotes/CheckinNoteServicesImpl.java

```diff
@@ -47,12 +47,6 @@ public CheckinNote read(@NotNull UUID id) {
         return checkinNoteRepository.findById(id).orElse(null);
     }
 
-    @Override
-    public Set<CheckinNote> readAll() {
```

Comment:
Getting a compilation failure on CheckinNoteServicesImplTest because this method is missing.

Reply:
removed it from tests and pushed the changes.